test/signalrclient-testhost/Connections/HubConnection.cs

```diff
@@ -26,5 +26,24 @@ public string ReturnMessage(string message)
         {
             return message;
         }
+
+        public async Task InvokeWithProgress(IProgress<int> progress)
+        {
+            for (int i = 0; i < 10; i++)
+            {
+                await Task.Delay(150);
```

Comment:
make it 10 - it does not matter in our case but we want tests to complete as quickly as possible. I would also send just 5 instead of 10 progress messages.